teams/hubturbo.md

```diff
@@ -33,25 +33,28 @@ permalink: /teams/hubturbo/
 
 * [x] Project set up - Week 1, Mon
 * [x] Domain understood - Week 1, Thu
-* [ ] Feature planned
+* [x] Feature planned - Week 3, Mon
 
 # Contributions
 
-### Choo Boon Yong Martin ([@m113225](https://github.com/m113225))
+### Choo Boon Yong Martin ([@m133225](https://github.com/m133225))
 
-* [ ] First PR submitted
-* [ ] First PR merged
+* [Pull Request #1191](https://github.com/HubTurbo/HubTurbo/pull/1191) (Merged)
+* [Pull Request #1193](https://github.com/HubTurbo/HubTurbo/pull/1193)
+* [Pull Request #1207](https://github.com/HubTurbo/HubTurbo/pull/1207)
+* [Pull Request #1208](https://github.com/HubTurbo/HubTurbo/pull/1208)
 
 ##### Contribution to project HT
 
+
 ##### Contribution to project Y
 
 ##### Technology expertise in tool Z
 
 ### Muhammad Rais Fathin Mudzakir ([@raisfathin](https://github.com/raisfathin))
 
-* [x] First PR submitted - Week 1, Thu
-* [ ] First PR merged
+* [Pull Request #1184](https://github.com/HubTurbo/HubTurbo/pull/1184) (Merged)
+* [Pull Request #1210](https://github.com/HubTurbo/HubTurbo/pull/1210)
```

Comment:
Include rough dates (e.g. Week 1, Thu) as we want to keep track of when something was done.